src/citrine/resources/data_concepts.py

```diff
@@ -388,6 +388,102 @@ def register_all(self, models: List[ResourceType], dry_run=False) -> List[Resour
         )
         return [self.build(obj) for obj in response_data['objects']]
 
+    def async_update(self, model: ResourceType,
+                     dry_run: Optional[bool] = False,
+                     wait_for_response: Optional[bool] = True,
+                     timeout: float = 2 * 60,
+                     polling_delay: float = 1.0) -> Optional[UUID]:
+        """
+        [ALPHA] Update a particular element of the collection with data validation.
+
+        Update a particular element of the collection, doing a deeper check to ensure that
+        the dependent data objects are still with the (potentially) changed constraints
+        of this change. This will allow you to make bounds and allowed named/labels changes
+        to templates.
+
+        Parameters
+        ----------
+        model: ResourceType
+            The DataConcepts object.
+        dry_run: bool
+            Whether to actually update the item or run a dry run of the update operation.
+            Dry run is intended to be used for validation. Default: false
+
```

Comment:
wait_for_response, timeout, and polling_delay need parameter docs

Reply:
done